### PR TITLE
#1783 Labels disappear on mouseover in Safari

### LINKED ISSUE #1783: Labels disappear on mouseover in Safari
After placing down a label and not giving it any ratings, if I move the mouse onto the label, it becomes invisible while the description tag still displays. The bug seems to go away if I click on the (invisible) label and give it a rating. Only happens on Safari (I tested on Safari, Firefox, and Chrome).

Screen recording: https://youtu.be/WeWBz_GCVBk

Picture:
Screen Shot 2019-07-09 at 12 08 46 PM

### PR DESCRIPTION
**Pull request by shokiami**
Resolves #1783 

This PR prevents the labels from disappearing on mouseover in Safari.

Before:
86502147-c3e9bf80-bd54-11ea-9b07-e2053ccc7b37

After:
87477043-48053800-c5dc-11ea-88bd-c0c89a0fd647

### CODE REVIEW

**Comment by misaugstad** [comment]
@ThatOneGoat could you test this PR on the various browsers (include Safari) on your mac? Then I'll test on my machine and review the code.

**Comment by ThatOneGoat** [comment]
Yup, will do!

**Comment by ThatOneGoat** [comment]
Looks like everything seems to be working as planned on safari and other browsers.

File: public/javascripts/SVLabel/src/SVLabel/label/Label.js [hunk 1]
```diff
@@ -599,7 +599,9 @@ function Label (svl, pathIn, params) {
         ctx.fillText(messages[0], labelCoordinate.x + padding.left, labelCoordinate.y + padding.top);
         if (hasSeverity) {
             ctx.fillText(severityMessage, labelCoordinate.x + padding.left, labelCoordinate.y + properties.tagHeight + padding.top);
-            ctx.drawImage(severityImage, labelCoordinate.x + padding.left + ctx.measureText(severityMessage).width + 5, labelCoordinate.y + 25, 16, 16);
+            if (properties.severity != undefined) {
```
Comment (misaugstad) [inline public/javascripts/SVLabel/src/SVLabel/label/Label.js:602]: I'm curious as to why this fixed it? What was going on here?

**Pull request merged**